src/MoBi.Assets/AppConstants.cs

```diff
@@ -1401,6 +1403,7 @@ public static class Captions
          public static readonly string RemoveCondition = "Remove condition";
          public static readonly string NewMatchTagCondition = "New match tag condition";
          public static readonly string NewNotMatchTagCondition = "New not match tag condition";
+         public static readonly string NewInContainerCondition = "New in container condition";
```

Comment:
Please take condition names within every caption in double quotes:
* _New "match tag" condition_ (or maybe _New "match tag"-condition_ or _New "Match tag" condition_ ?)
* _New "not match tag" condition_ (or ...)
* _New "in container" condition_ (or ...)

Reply:
Sure. I just followed what was there before
This will break ranorex tests